Sources/PostgresKit/PostgresDialect.swift

```diff
@@ -37,4 +33,9 @@ public struct PostgresDialect: SQLDialect {
     public var enumSyntax: SQLEnumSyntax {
         .typeName
     }
+
+    public var triggerSyntax: SQLTriggerSyntax {
+        return .init(create: [.supportsForEach, .postgreSQLChecks, .supportsCondition, .conditionRequiresParentheses, .supportsConstraints],
```

Comment:
Vapor style aligns indentation on new lines, e.g.:
```swift
Foo(
    bar: ...,
    baz: ...
)
```

Reply:
I'm not sure what you're asking me to do here.

Reply:
@grosch
```swift
    return init(
        create: [ ... ],
        drop: [ ... ]
    )
````

Reply:
Thanks, done.